### PR TITLE
OpenStack: Update CI flavor for vexxhost

### PR DESCRIPTION
**Pull request by mandre**
The previous flavor caused job to run in the wrong aggregate.

### CODE REVIEW

**Comment by pierreprinetti** [comment]
/lgtm
/approve
moar power!

/hold
Cancel the hold when ready!

**Comment by mandre** [comment]
[release-openshift-ocp-installer-e2e-openstack-serial-4.7](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/16665/rehearse-16665-release-openshift-ocp-installer-e2e-openstack-serial-4.7/1369590413496684544) succeeded with the new flavor.

The other jobs are either still running, or failed because they couldn't get a boskos lease.

Once this patch merges, I'll submit another patch to bump our vexxhost quota from 3 to 9. We can't bump quota while we're still using the wrong flavor.

**Review by droslean — APPROVED** [review]

**Pull request merged**
